### DISCUSSION TITLE
Farmer running and connected to peers, but not syncing

### DISCUSSION THREAD

**Discussion by swilkinson1** [Chia-Network/chia-blockchain#4966] [Support]
I have a farmer running (farmer, full node, harvester, wallet, daemon) and connected to multiple peers, but can't get synced. Lost sync overnight, and now this morning it won't sync back up. Port 8444 is open and set up for port forwarding. Running `chia version` 1.1.6.dev0 on Ubuntu 20.04.2.

Output from `chia show -sc`:
Current Blockchain Status: Not Synced. Peak height: 278687
      Time: Fri May 14 2021 08:11:15 UTC                  Height:     278687

Estimated network space: 4.423 EiB
Current difficulty: 304
Current VDF sub_slot_iters: 112721920
Total iterations since the start of the blockchain: 899949780394

  Height: |   Hash:
   278687 | 891792cff2c48d1b648817a28d7e7c147e6bf5ee541e3355c9ac08a5f7b4e0b4
   278686 | 496ab3dc5131d339f3e6237f4cb792864167f64e5ab266ba7062ba770c60a3d7
   278685 | 1ec7d0815db6ef180f1a3d1a0185bfe2b4d3b5d82d30990db0bd3e6b695ef067
   278684 | 1d2cedb5f357b47b2c3a5253917911e0a476c83e161b013f979fd0b7878dcda8
   278683 | 3d2168ffb56b65595b1d88d99f5006cc06b3d31fd1d3e9d1860c08326444d291
   278682 | 7782ad9204352353d4b98a11a0ffce127e4d936953203b5ea5ff9350d9d18b56
   278681 | 8f1874501055338cce05b0aa35cfa511297a787fbbe32f66234ed8a5b876be97
   278680 | 47b2c3ebb66432fba4e1064e347cbc089b530ce744e17d1206224999b76fe25c
   278679 | 65d27ae4a163e9f05a7e93fd25d7ebbbf6c30f21e637cd58a5a31a1505afe7b1
   278678 | 84f603e8fa662451ead412e034927ee880dd5274cc79fb9005bd5e4bd2661227

Connections:
Type      IP                                     Ports       NodeID      Last Connect      MiB Up|Dwn
WALLET    127.0.0.1                              34426/8449  ed2a1260... May 14 11:43:49      0.0|0.0    
FARMER    127.0.0.1                              34434/8447  38946635... May 14 11:43:51      0.0|0.0    
FULL_NODE 84.193.35.3                            37342/8444  1ef55227... May 14 12:44:27      0.0|0.0    
                                                 -SB Height:      297    -Hash: 02435862...
FULL_NODE 73.87.69.179                           52375/8444  5b5fc544... May 14 12:56:39     38.9|0.0    
                                                 -SB Height:     7321    -Hash: 89d150ba...
FULL_NODE 149.34.5.4                             51983/8444  dccf94f0... May 14 12:58:48     46.4|0.0    
                                                 -SB Height:        0    -Hash:  Info...
FULL_NODE 66.25.26.18                            65047/8444  bb1b66ca... May 14 12:57:04     33.9|0.0    
                                                 -SB Height:   259959    -Hash: 43b77e86...
FULL_NODE 186.250.161.61                         57068/8444  70698828... May 14 12:58:01      0.0|0.0    
                                                 -SB Height:   264970    -Hash: 2efc6ba0...
FULL_NODE 88.80.148.241                          49890/8444  3ce8e907... May 14 12:44:27      0.0|0.0    
                                                 -SB Height:        0    -Hash:  Info...
INTRODUCER introducer-va.chia.net                  8444/8444  0b1ad21d... May 14 12:46:22      0.0|0.0    
FULL_NODE 118.200.54.16                           8444/8444  c6efb1b1... May 14 12:49:48      0.0|0.0    
                                                 -SB Height:   200160    -Hash: 084453bf...
FULL_NODE 5.216.181.83                           49868/8444  7a434aff... May 14 12:58:12      0.0|0.0    
                                                 -SB Height:      160    -Hash: 853946df...

Output from `chia farm summary`:
Farming status: Not synced or not connected to peers

**Comment by cihansardaslar** [comment]
Same here my 8444 port is open but there is a error with 8449. And i tried everything to open port 8449 but i cant open it. Idk why. and also i have 9 full node connections, one farmer it says and one wallet connection. but its my nearly third day and still not sync. my log is like this

2021-05-14T16:05:09.041 full_node full_node_server        : ERROR    Exception:  <class 'concurrent.futures._base.CancelledError'>, closing connection {'host': '127.0.0.1', 'port': 8449}. Traceback (most recent call last):
  File "chia\server\server.py", line 531, in api_call
  File "asyncio\tasks.py", line 435, in wait_for
concurrent.futures._base.CancelledError

2021-05-14T16:05:10.544 full_node full_node_server        : ERROR    Exception:  <class 'concurrent.futures._base.CancelledError'>, closing connection {'host': '127.0.0.1', 'port': 8449}. Traceback (most recent call last):
  File "chia\server\server.py", line 531, in api_call
  File "asyncio\tasks.py", line 435, in wait_for
concurrent.futures._base.CancelledError

2021-05-14T16:05:12.063 full_node full_node_server        : ERROR    Exception:  <class 'concurrent.futures._base.CancelledError'>, closing connection {'host': '127.0.0.1', 'port': 8449}. Traceback (most recent call last):
  File "chia\server\server.py", line 531, in api_call
  File "asyncio\tasks.py", line 435, in wait_for
concurrent.futures._base.CancelledError

**Comment by swilkinson1** [comment]
I am seeing these errors in my logs:

2021-05-14T11:29:57.219 full_node full_node_server        : INFO     Connection closed: 66.75.0.206, node id: b23c644d661a25686978ae004d392f9e03312b9ca340ff99f34ea8fd2883047c
2021-05-14T11:29:57.219 full_node full_node_server        : WARNING  Banning 66.75.0.206 for 600 seconds
2021-05-14T11:29:57.220 full_node chia.full_node.full_node: INFO     peer disconnected {'host': '66.75.0.206', 'port': 8444}
2021-05-14T11:29:57.221 full_node chia.full_node.full_node: ERROR    Error with syncing: <class 'RuntimeError'>Traceback (most recent call last):
  File "/home/ubuntu/chia-blockchain/chia/full_node/full_node.py", line 628, in _sync
    raise RuntimeError(f"Weight proof did not arrive in time from peer: {weight_proof_peer.peer_host}")
RuntimeError: Weight proof did not arrive in time from peer: 66.75.0.206

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
